test/openshift/e2e/sequential/1-034_validate_custom_roles/02-patch-subscription.yaml

```diff
@@ -3,9 +3,14 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
 - script: |
-    if ! [ -z $CI ]; then
 
-      oc patch -n openshift-operators subscription `subscription=gitops-operator- && oc get subscription --all-namespaces | grep $subscription | head -1 | awk '{print $2}'`\
+    if ! [ -z $NON_OLM ]; then       
+        oc patch deployment gitops-operator-controller-manager -n gitops-operator-system \
+          -p '{"spec":{"template":{"spec":{"containers":[{"name":"manager","env":[{"name":"CONTROLLER_CLUSTER_ROLE","value":"custom-argocd-role"},{"name":"SERVER_CLUSTER_ROLE", "value":"custom-argocd-role"}]}]}}}}'
+     
+    elif ! [ -z $CI ]; then    
+      
+      oc patch -n openshift-operators subscription $sub \
```

Comment:
Any reason to change the command here for CI?

Reply:
It has something to do with the name of the subscription, on the CI and in the cluster then subscription name can be different. However it is not related to my change - I have just added elif for the third non-olm condition

Reply:
It seems that the code got updated by mistake then.. 

Github PR shows that the line
```
oc patch -n openshift-operators subscription `subscription=gitops-operator- && oc get subscription --all-namespaces | grep $subscription | head -1 | awk '{print $2}'`\
```

was updated to 
```
oc patch -n openshift-operators subscription $sub \`
```
for CI case.